README.md

```diff
@@ -10,7 +10,7 @@ To get started, clone the repo, follow the steps in `Initial Setup`, below to se
 
 *This assumes you have [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/install/) already installed.*
 ```
-docker compose --env-file .env up
+docker compose up
```

Comment:
The explicit env-file param isn't necessary, since we're referencing the env file in docker-compose.yaml